### PR TITLE
AML-2237 Updated transfers dashboard to request transfer requests from commitments api

### PR DESCRIPTION
**Pull request by elliotritchie**

### CODE REVIEW

File: src/SFA.DAS.EAS.Account.Worker.UnitTests/Foo.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using NUnit.Framework;
```
Comment (robinoconnor) [inline src/SFA.DAS.EAS.Account.Worker.UnitTests/Foo.cs]: Is this needed? Looks like its a placeholder that never got replaced.

File: src/SFA.DAS.EAS.Api/Web.config [hunk 2]
```diff
@@ -194,7 +194,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.IdentityModel.Clients.ActiveDirectory" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.16.0.14" newVersion="3.16.0.14" />
+        <bindingRedirect oldVersion="0.0.0.0-3.13.8.999" newVersion="3.13.8.999" />
```
Comment (robinoconnor) [inline src/SFA.DAS.EAS.Api/Web.config]: What's the reason for the downgrading of versions?

File: src/SFA.DAS.EAS.DbMaintenance.UnitTests/app.config [hunk 3]
```diff
@@ -28,7 +28,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.IdentityModel.Clients.ActiveDirectory" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.16.0.14" newVersion="3.16.0.14" />
+        <bindingRedirect oldVersion="0.0.0.0-3.13.8.999" newVersion="3.13.8.999" />
```
Comment (robinoconnor) [inline src/SFA.DAS.EAS.DbMaintenance.UnitTests/app.config]: Again why are we downgrading the version?

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs [hunk 4]
```diff
@@ -51,7 +50,7 @@ public ActionResult Latest(GetLatestActivitiesQuery query)
             {
                 _logger.Warn(ex, "Failed to get the latest activities.");
 
-                return Content(ControllerConstants.ActivitiesUnavailableMessage);
+                return Content("Activities are currently unavailable.");
```
Comment (robinoconnor) [inline src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Are we not using constants anymore?

File: src/SFA.DAS.EAS.Account.Worker.UnitTests/Foo.cs (hunk 1 above)
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Account.Worker.UnitTests/Foo.cs]: Yeah, if there's no tests in the assembly the build fails so I added a placeholder. I'd delete the project but it's being re-purposed by Ian on his branch.

File: src/SFA.DAS.EAS.Api/Web.config (hunk 2 above)
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Api/Web.config]: Fixed

File: src/SFA.DAS.EAS.DbMaintenance.UnitTests/app.config (hunk 3 above)
Comment (elliotritchie) [reply src/SFA.DAS.EAS.DbMaintenance.UnitTests/app.config]: Fixed

File: src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs (hunk 4 above)
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/Controllers/ActivitiesController.cs]: Fixed

**Pull request merged**